azkaban-web-server/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -1065,6 +1065,7 @@ private void ajaxUpdateProperty(final HttpServletRequest req,
       if (!HttpRequestUtils.hasPermission(this.userManager, user, Type.ADMIN)) {
         ret.put("error", String.format("User %s doesn't have ADMIN permission for updating "
             + "property", user));
+        return;
```

Comment:
For your reference, the ajaxRampActions method also rely on hasPermission to check, and will just ignore all non-admin ppl:
https://github.com/azkaban/azkaban/blob/master/azkaban-web-server/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java#L1174